### PR TITLE
Ujson upgrade

### PR DESCRIPTION
**Pull request by ZAlfonse**
Upgrading to usjon for performance reasons, falling back on old json and putting out a warning.

### CODE REVIEW

File: savory_pie/django/haystack_field.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
-import json
+try:
+    import ujson as json
+except ImportError:
+    import warnings
+    warnings.warn('Using plain JSON instead of uJSON, performance may be degraded.')
```
Comment (darrentorpey) [inline savory_pie/django/haystack_field.py]: Minor: `from warnings import warn` (then just `warn('...')` below that may be more performant. I'm not sure how much memory bloat you get from importing the whole library. Just a thought.

**Comment by arianamiri** [comment]
You might want to add a line to read me encouraging the installation of ujson.  Otherwise looks fine.

**Comment by nsheaps** [comment]
I agree with @arianamiri 's comment

File: savory_pie/django/haystack_field.py (hunk 1 above)
Comment (cclough2) [inline savory_pie/django/haystack_field.py]: Agreed. Usually you really should only import exactly what you need.

**Comment by cclough2** [comment]
Seems fine

**Pull request merged**